spec/paperclip/content_type_detector_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'spec_helper'
 
 describe Paperclip::ContentTypeDetector do
+  it 'returns a meaningful content type for open xml spreadsheets' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.